### PR TITLE
[MDS-4755] core updates major projects table

### PR DESCRIPTION
**Pull request by areyeslo**
## Objective 

[MDS-4755](https://bcmines.atlassian.net/browse/MDS-4755)

- Add filters for MRC and EMLI Project Lead.
- Click on Open button redirects to Overview Page.

image

### CODE REVIEW

File: services/core-api/app/api/projects/project/resources/project.py [hunk 1]
```diff
@@ -276,11 +285,36 @@ def _apply_filters_and_pagination(self, args):
...
+            if len(project_lead_conditions) == 2:
+                conditions.append({'or': project_lead_conditions})
+
         projects_query = apply_filters(query, conditions)
 
         # Apply sorting
         if sort_model and args['sort_field'] and args['sort_dir']:
-            sort_criteria = [{'model': sort_model, 'field': args['sort_field'], 'direction': args['sort_dir']}]
+            sort_criteria=[]
+            if sort_model == 'Party':
```
Comment (BenHarrington) [inline services/core-api/app/api/projects/project/resources/project.py:313]: I notice when using the column sorter that all entries which do not have a project_lead set do not get returned in the sorted result, is that to be expected?

**Review by BenHarrington — COMMENTED** [review]
Other than my one comment, this appears to look ready to go.

File: services/core-api/app/api/projects/project/resources/project.py (hunk 1 above)
Comment (areyeslo) [reply services/core-api/app/api/projects/project/resources/project.py:313]: Fixed.

**Review by BenHarrington — APPROVED** [review]
Thanks for making that change, looks good!

**Pull request merged**